web/src/copy_messages.ts

```diff
@@ -52,6 +53,48 @@ function construct_recipient_header($message_row: JQuery): JQuery {
         .replace(/\s$/, "");
     return $("<p>").append($("<strong>").text(message_header_content));
 }
+
+function get_first_and_last_selected_elements(): {
```

Comment:
This name is confusing, in that it seems to never return an actual element in the message, but instead something constructed virtually?

Reply:
Actually this abstraction seems weird; it seems like the main determination this is making is the determination of the `$fragment_children` elements; would it make sense to just do that logic separately for the start and end of the selection?